src/components/TransactionList/TransactionRow/TransactionRowTs.ts

```diff
@@ -412,11 +412,12 @@ export class TransactionRowTs extends Vue {
     @Watch('transaction', { immediate: true })
     private async fetchTransaction() {
         if (
-            this.transaction instanceof AggregateTransaction &&
-            this.transaction.type === TransactionType.AGGREGATE_BONDED &&
-            this.hasMissSignatures
+            (this.transaction instanceof AggregateTransaction &&
+                this.transaction.type === TransactionType.AGGREGATE_BONDED &&
+                this.hasMissSignatures) ||
+            this.transaction instanceof AggregateTransaction
```

Comment:
It seems like this is equivalent to the following, isn't it?
```
if (this.transaction instanceof AggregateTransaction) 
```

Reply:
the second case ignores checking for missing signature.

Reply:
This `if` check will result in `true` if `this.transaction instanceof AggregateTransaction` is `true` (because of the second check - right-hand side of the 'or' operator), right? If so what's the point of having the first part of the `||` here? In any case when the transaction is instance of AggregateTransaction then it will return true then this is equivalent to the following if that makes sense?
```
if (this.transaction instanceof AggregateTransaction) 
```